apps/site/pages/docs/api-extensions/extending-queries-using-fragments.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: ' Extending queries using fragments'
+sidebar_label: 'API Extensions'
+---
+
+import { Callout, Step, Steps } from 'nextra-theme-docs'
+import { ExtendableQueryTable } from './../../../components/ExtendableQueryTable'
+
+<header>
+
+# Extending queries using fragments
+
+</header>
+
+After [defining the new fields exposed by the FastStore API Extension](/docs/api-extension/extending-api-schema), the next step is to specify where these fields will be utilized within GraphQL fragments.
+
+Fragments in GraphQL are components that enhance query functionality. FastStore associates these fragments with the existing queries used on its pages to incorporate the newly exposed fields.
```

Comment:
I wouldn't call them "components" as people might confuse them with React components.

Reply:
Good point! I added the link to the [Fragment's definition](https://graphql.org/learn/queries/#fragments) on the GraphQL page.